cluster-provision/cli/cli

```diff
@@ -190,6 +190,8 @@ if [ "$COMMAND" == "provision" ] ; then
   docker exec ${VM_CID} /bin/bash -c "scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i vagrant.key -P 22 /manifests/ceph/*.yaml vagrant@192.168.66.101:/tmp/ceph"
   docker exec ${VM_CID} /bin/bash -c "ssh.sh mkdir -p /tmp/cna"
   docker exec ${VM_CID} /bin/bash -c "scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i vagrant.key -P 22 /manifests/cna/*.yaml vagrant@192.168.66.101:/tmp/cna"
+  docker exec ${VM_CID} /bin/bash -c "ssh.sh mkdir -p /tmp/cni"
```

Comment:
Do we really need to refactor the way we push manifests to the image? If possible, please leave the re-factoring from this PR and allow forwarding in the simplest way possible to unblock CI.